mopidy_local/ext.conf

```diff
@@ -44,3 +44,6 @@ use_artist_sortname = false
 # a list of file names to check for when searching for external album
 # art; may contain UNIX shell patterns, i.e. "*", "?", etc.
 album_art_files = *.jpg, *.jpeg, *.png
+
+# minimum required size, in pixels, for an image to be picked up as an album cover
+min_art_size = 32
```

Comment:
Part of me think it's not totally obvious this means the width and height, and not the total pixel count. We should also say this new feature is disabled by setting the value to 0, or making it optional. And arguably 0 should be the default value else this is a breaking change.

And I think we should call it `album_art_min_size` to be more in line with the existing setting names.

Reply:
maybe "dimension" would be less ambiguous than "size" ?

Reply:
Perhaps just make the comment clearer. And it needs a corresponding entry in the configuration section of the readme

Reply:
comment updated (and name also)
It's already documented in the readme too

btw, two tests added